internal/connect/errors.go

```diff
@@ -36,11 +36,12 @@ type ZypperError struct {
 	Commmand []string
 	ExitCode int
 	Output   []byte
+	Err      error
```

Comment:
This makes ZypperError the same inside as ExecuteError. I'm fine with it as long as it helps debug https://bugzilla.suse.com/show_bug.cgi?id=1200803 , but you may want to merge the types or add a comment why they stay separate.
Also, I don't know Golang.